### DISCUSSION TITLE
Unable to expand results

### DISCUSSION THREAD

**Discussion by CovertCode** [pocketbase/pocketbase#4249] [Q&A] [answered]
I have 2 collections:
1) `vendors` which is auth collection
2) `category`

In vendor I have multiple relation field as `categories` where I insert category.
![image](https://github.com/pocketbase/pocketbase/assets/6436693/cd686cbf-e2e0-423d-9d98-ca3d63b1e4cb)

Now when I try to get expanded results I am not getting any expanded field.
```
const record = await locals.pb.collection('vendors').getOne(z.model.id, {
    expand: 'categories',
});
```

Instead I am getting same result if I don't use `expand`.
I am fetching this `vendor` data from the authenticated user cookie from astro.

**Comment by ganigeorgiev** [comment, marked as answer]
Make sure that your client request satisfies the View API rule of the "category" collection.

**Reply by CovertCode** [reply]
You're right. I forgot it is all set to admin.
Fixed it by setting `@request.auth.id != ""` in `category` collection.

Is it possible to set api rule such that user who have `category.collectionId` in they `vendor.categories`?
Example: Lets say there are 10 records under `category` collection. In that single `vendor.categories` only 4 category items are in that field.
When that vendor make query to get category list then only the assigned category are shown while other are not.
Is it possible?

**Reply by ganigeorgiev** [reply]
I don't understand your follow up question. Please provide more details about the available "users" collection fields and their types.
